src/routes/opening/components/Footer.tsx

```diff
@@ -75,7 +75,13 @@ export const ContinueFooter = ({
       variant="contained"
       data-testid="continue-btn"
     >
-      Get started
+      {continueButtonDisabled ? (
+        <>
+          <Loader size="xs" color="secondaryLight" /> Loading your safe
```

Comment:
We need a small gap between the loader and the text. Rn they are right next to each other.